pim-tool-back/src/main/java/vn/elca/training/model/entity/Employee.java

```diff
@@ -0,0 +1,37 @@
+package vn.elca.training.model.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+
+/**
+ * @author thomas.dang
+ */
+@Entity
+@Getter
+@Setter
+@Table(name = "EMPLOYEE")
+public class Employee {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "ID", nullable = false)
+    private Long id;
```

Comment:
[S] All entities have `id` and `version`. You should introduce an abstract class.